kernel/kernel-impl/src/main/java/org/sakaiproject/content/impl/DbContentService.java

```diff
@@ -1941,7 +1941,7 @@ public void removeResource(ContentResourceEdit edit, boolean removeContent)
 							String filePath = ((BaseResourceEdit) edit).m_filePath;
 							log.info("Removing resource ("+edit.getId()+") content: "+bodyPath+" file:"+filePath);
 
-							String statement = "SELECT COUNT(FILE_PATH) FROM "+resourceTableName+" WHERE FILE_PATH = ?;";
+							String statement = "SELECT COUNT(FILE_PATH) FROM "+resourceTableName+" WHERE FILE_PATH = ?";
```

Comment:
@jesusmmp can we also move this statement into the SqlService?

Reply:
Of course! I'll fix it tomorrow